js/views/modals/purchase/Purchase.js

```diff
@@ -250,18 +251,20 @@ export default class extends BaseModal {
   disableModerators() {
     this.getCachedEl('#purchaseModerated').prop('checked', false);
     this.order.moderated = false;
+    this.moderators.deselectOthers();
     this.order.set('moderator', '');
   }
 
-  toggleModeration(bool) {
+  moderationOn(bool) {
```

Comment:
The name was changed here so it's more obvious what it's doing, and whether the value passed in should be true or false.